### ISSUE TITLE
Setting images causes error

### ISSUE THREAD

**Issue by nickasd** [destinationstransfers/passkit#42]
First of all, thank you for maintaining this useful library.

The README says that in order to load some images:
```
pass.images.icon = iconFilename;
pass.icon(iconFilename);
await pass.images.loadFromDirectory('./images');
```
I understand that each of these lines can be used alone (and you don't have to call the first or second line followed by the third one), but when I use `template.images.loadFromDirectory(directoryname)` followed by `pass.images.icon = filename`, I get the following error:
```
error: Unhandled Rejection at: Promise {
   <rejected> { Error: EISDIR: illegal operation on a directory, read errno: -21, code: 'EISDIR', syscall: 'read' } } reason: Error: EISDIR: illegal operation on a directory, read
```
What am I doing wrong?

**Comment by nickasd** [comment]
Also it is confusing that the pass file is created anyway. In my opinion, if an error occurs, the pass file should not be created and the `error` event should be triggered. Maybe this is not happening because you're not handling that promise's rejection.

**Comment by nickasd** [comment]
There is a similar issue when the provided .pem file is not valid: an unhandled rejection exception is thrown. Could this be fixed as well?

**Comment by tinovyatkin** [comment]
@nickasd All these issues were fixed in 6.0 released today. Thank you!
